aiida_quantumespresso/workflows/pdos.py

```diff
@@ -159,6 +162,17 @@ def validate_projwfc(value, _):
     jsonschema.validate(value['parameters'].get_dict()['PROJWFC'], get_parameter_schema())
 
 
+def validate_fermi_energy_range(value, _):
```

Comment:
@sphuber I added "_" in the inputs of the validator. Otherwise one would get a warning that a validator with just one input is deprecated. Please correct me if I am wrong.

Reply:
That is the correct solution indeed, we do this routinely.